### ISSUE TITLE
Getting started images are missing alt text

### ISSUE THREAD

**Issue by martinohanlon** [RaspberryPiFoundation/picozero#89]
**Describe the bug**
The images in the getting started guide of the documentation are missing alt text descriptions.

**Comment by martinohanlon** [comment]
Resolved in v0.4.1
